libs/net/dal/Configuration/SourceConfiguration.cs

```diff
@@ -21,6 +21,7 @@ public override void Configure(EntityTypeBuilder<Source> builder)
         builder.Property(m => m.DisableTranscribe).IsRequired();
         builder.Property(m => m.UseInTopics).IsRequired();
         builder.Property(m => m.Configuration).IsRequired().HasColumnType("jsonb").HasDefaultValueSql("'{}'::jsonb");
+        builder.Property(m => m.IsCBRASource).IsRequired(false);
```

Comment:
According to your entity it is required.

Reply:
I will update. thanks

Reply:
updated. thanks